src/AzureRMTools.ts

```diff
@@ -162,7 +162,8 @@ export class AzureRMTools {
                             return;
                         }
 
-                        // Telemetry for template opened
+                        // Template for template opened
```

Comment:
Telemetry for template opened?

Reply:
I fixed that, must have slipped in through a merge.  Thx.